packages/dashboard/src/components/lifts-app.tsx

```diff
@@ -25,33 +26,40 @@ export const LiftsApp = createMicroApp('Lifts', () => {
       return;
     }
 
-    buildingMap?.lifts.map((lift, i) => {
-      const sub = rmf.getLiftStateObs(lift.name).subscribe((liftState) => {
-        setLiftTableData((prev) => {
-          return {
-            ...prev,
-            [lift.name]: [
-              {
-                index: i,
-                name: lift.name,
-                currentFloor: liftState.current_floor,
-                destinationFloor: liftState.destination_floor,
-                doorState: liftState.door_state,
-                motionState: liftState.motion_state,
-                lift: lift,
-                onRequestSubmit: (_ev, doorState, requestType, destination) => {
-                  return rmf?.liftsApi.postLiftRequestLiftsLiftNameRequestPost(lift.name, {
-                    destination,
-                    door_mode: doorState,
-                    request_type: requestType,
-                  });
+    buildingMap?.lifts.map(async (lift, i) => {
+      try {
+        const { data } = await rmf.liftsApi.getLiftHealthLiftsLiftNameHealthGet(lift.name);
+        const { health_status } = data;
+        const sub = rmf.getLiftStateObs(lift.name).subscribe((liftState) => {
+          setLiftTableData((prev) => {
+            return {
+              ...prev,
+              [lift.name]: [
+                {
+                  index: i,
+                  name: lift.name,
+                  opMode: health_status ? health_status : 'N/A',
```

Comment:
I think we'll need to remap the text, https://github.com/open-rmf/rmf-web/blob/main/packages/api-server/api_server/models/health.py#L10, from Healthy, Unhealthy, Dead to ONLINE, UNSTABLE, OFFLINE, with a similar styling with the door status (colored font, uppercase)

Reply:
![image](https://github.com/open-rmf/rmf-web/assets/5383623/657e1472-a806-4196-b2a6-bb9ff239ef3d)

Reply:
Nice!
Done here 7bf9c9673c38fd08fa38d14f1e5e81b2b178a5b0